nilearn/datasets/struct.py

```diff
@@ -0,0 +1,334 @@
+# *- encoding: utf-8 -*-
+"""
+Downloading NeuroImaging datasets: structural datasets
+"""
+import warnings
+import os
+import numpy as np
+from sklearn.datasets.base import Bunch
+
+from utils import _get_dataset_dir
+from utils import _fetch_files
+from utils import _get_dataset_descr
```

Comment:
"from .utils" please. This will not work in python 3: there are no implicit absolute imports.